packages/orbit-components/.storybook/manager.js

```diff
@@ -1,10 +1,11 @@
-import { create } from "@storybook/theming"
+// @flow
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Instead of adding disable directives in `.storybook` you can include `**/.storybook` in [this list](https://github.com/kiwicom/orbit/blob/7bd4ab3cc74b7c42a6d3817d6712865709a7e1bd/config/eslintJS.js#L22-L30). Dealing with ESLint can sometimes be so tedious… 😓

Reply:
yes, i know, it was first thing which came to my mind, but later i noticed that there are already eslint disables in these files, so i picked the easy way out :D